app/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 29
+
+    defaultConfig {
+        applicationId "org.dsandler.apps.markers"
+        minSdkVersion 14
+        targetSdkVersion 29
```

Comment:
To jump [`targetSdkVersion` from `19` to `29`](https://developer.android.com/distribute/best-practices/develop/target-sdk ), looking through the changes it doesn't *seem* like Markers was modified to make use of run-time permissions and scoped storage?

My experience with the current version has Markers saving directly to a location on internal storage.  (I haven't actually tried compiling and running; I may be mistaken.)

Regardless, thanks for the effort to help revive this app!

Reply:
Yeah you're right, -2 this change for now.